src/main.rs

```diff
@@ -55,6 +54,22 @@ enum Opt {
         #[clap(short, long)]
         google_sheet: Option<String>,
     },
+
+    /// list all repositories in the given organization and the number of
+    /// Pull Requests created in the last 30 days
+    RepoParticipants {
+        /// name of GitHub organization to analyze
+        #[clap(short, long)]
+        org: String,
+
+        /// name of GitHub organization to analyze
+        #[clap(short, long)]
+        repo: String,
+
+        /// Verbose mode (-v, -vv, -vvv, etc.)
+        #[clap(short, long, parse(from_occurrences))]
+        verbose: u8,
```

Comment:
I'm personally a bit curious why we're keeping `verbose` on either command when they go unused for the moment.

Reply:
we could remove it :)